src/linear_algebra/dual.lean

```diff
@@ -570,6 +583,26 @@ begin
     exact is_basis.to_dual_equiv _ hB.1 },
 end
 
+open finite_dimensional
+
+@[simp]
+lemma findim_dual_annihilator_comap_eq {Φ : subspace K (module.dual K V)} :
+  findim K Φ.dual_annihilator_comap = findim K Φ.dual_annihilator :=
+begin
+  rw [submodule.dual_annihilator_comap,
+      show module.dual.eval K V = vector_space.eval_equiv.to_linear_map, by refl],
```

Comment:
This looks like it probably ought to be a lemma

Reply:
Done!